src/main/java/org/randywebb/wlts/beans/Teacher.java

```diff
@@ -23,7 +23,7 @@
         return fromArray(array, new ArrayList<Teacher>(), Teacher.class);
     }
 
-    /** Default constructor */
+    /** Default constructor. */
     public Teacher() {
```

Comment:
I don't usually bother defining a default constructor unless I'm going to add logic to it.

Reply:
This needed a default constructor for some reason (I don't add default constructors without a reason either)

Reply:
👍